nrfconnect-chip/Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN set -x \
     #
     && chmod 777 /tmp && apt-get update \
     && apt-get -y install autoconf \
-    && apt-get -y install --no-install-recommends sudo make g++ g++-multilib \
+    && apt-get -y install --no-install-recommends sudo make g++ gcc-multilib-arm-linux-gnueabi gcc-multilib-arm-linux-gnueabihf \
```

Comment:
Actually, you probably don't need both, but I didn't spend the time to check which one is needed.